src/core/lib/iomgr/tcp_client_posix.c

```diff
@@ -86,8 +86,9 @@ static grpc_error *prepare_socket(const grpc_resolved_address *addr, int fd,
   err = grpc_set_socket_cloexec(fd, 1);
   if (err != GRPC_ERROR_NONE) goto error;
   if (!grpc_is_unix_socket(addr)) {
-    err = grpc_set_socket_low_latency(fd, 1);
-    if (err != GRPC_ERROR_NONE) goto error;
+    /* We're going to ignore the potential error on this one, because it's not
+       a fatal problem per-se. */
+    grpc_set_socket_low_latency(fd, 1);
```

Comment:
The environment behaves really strongly the same as any other Linux. The only difference I've managed to spot is the result of uname.